### DISCUSSION TITLE
reComputer J30 Series, J40 Series | J401 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#504] [Q&A]
# reComputer J30 Series, J40 Series | J401 | Seeed Studio Wiki

Flash JetPack to reComputer J4012 (J401 carrier board)

https://wiki.seeedstudio.com/reComputer_J4012_Flash_Jetpack/

**Comment by jinchenglee** [comment]
Is J4012 coming with 100 TOPS or 70 TOPS? What is listed here is conflicting with the table: https://www.seeedstudio.com/reComputer-J4012-p-5586.html.

**Reply by MatthewJeffson** [reply]
Hi, the J4012 is equipped with 100TOPS computing power, and it indeed seems like the product page is not clear enough. I will inform my colleague about it.
We have [the comparison for all the NVIDIA Jetson Devices and carrier boards](https://files.seeedstudio.com/products/NVIDIA/NVIDIA-Jetson-Devices-and-carrier-boards-comparision.pdf) in the [NVIDIA Jetson devices collection page](https://wiki.seeedstudio.com/NVIDIA_Jetson/) and hopefully you find it more clearer.
Regards

**Comment by pvonhahn** [comment]
Can I flash the J4012 using the NVIDIA SDK Manager if I want? If not, why not?

**Reply by xzxcessarr** [reply]
Hello, the answer is well, you can use the NVIDIA SDK Manage, because all of our carrier board is match the officially designed board of NVIDIA, and we recommand you to follow our tutorial in wiki to set your device

**Comment by knoffelcut** [comment]
I have a J401 carrier board with a Jetson Orin Nano module and I'm struggling to get the board to boot.

If connected to a HDMI display and power on, should:
- The display show a boot screen (similar to the Jetson Orin Nano dev kit)?
- The fan turn on?

In my case I observe both those on the Jetson Orin Nano dev kit but not on the J401 carrier board.

**Comment by tbanetwork** [comment]
There has been an Update from NVIDIA on how to fix the DTB for JP 5.1.2:

--- a/firewall/tegra234-mb2-bct-scr-p3767-0000.dts
+++ b/firewall/tegra234-mb2-bct-scr-p3767-0000.dts
@@ -5,6 +5,11 @@
/ {
tfc {
+ reg@322 { /* GPIO_M_SCR_00_0 */
+ exclusion-info = <2>;
+ value = <0x38009696>;
+ };
+
reg@5138 { /* CBB_CENTRAL_CBB_FIREWALL_QSPI0_BLF, READ_CTL */
exclusion-info = <2>;
value = <0x00100009>;

Maybe you should update the guide.

**Reply by tbanetwork** [reply]
Also see updated PDF Document: https://docs.nvidia.com/jetson/archives/r35.4.1/ReleaseNotes/Jetson_Linux_Release_Notes_r35.4.1.pdf

**Comment by lucasvickers** [comment]
The package comes with 2 SMA wifi antennas, but I don't see anywhere for them to connect. What am I missing?

**Comment by yalexx** [comment]
Can we get updated instructions for Jetpack 6

**Reply by costasvav** [reply]
Also looking for JP6.0.  Too many benefits to not upgrade.

**Comment by yalexx** [comment]
What bootloader do i have to select ?

**Comment by BrettOJ** [comment]
Anyone know how to flash with JP 6.  I need to flash because JP 5 does not support latest version of Pytorch and as such fails to import some FastAI libraries :-( .  I have bought 2 of these (SG$ 2600) and can't get either to work :-(
